src/logger.ts

```diff
@@ -0,0 +1,41 @@
+import * as winston from "winston";
+import { Logger } from "winston";
```

Comment:
Prefer `const { Logger } = winston;` to avoid the double import.

Reply:
Alternatively 

```suggestion
export const Logger = winston.Logger;
```

to achieve 2 goals with one line of code.

Reply:
Logger is not declared as part of winston type, it wont compile.

Reply:
That suggests the `import * as winston` is misleading then. I wrote `export const Logger = ` before, but it seems I meant `export type Logger =`